### ISSUE TITLE
homebrew-julia uses deprecated Resource#sha1

### ISSUE THREAD

**Issue by jenshnielsen** [staticfloat/homebrew-julia#216]
Running `brew outdated` results in:

```
Warning: Calling Formula.sha1 is deprecated!
Use Formula.sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/gmp4-julia.rb:9:in `<class:Gmp4Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling SoftwareSpec#sha1 is deprecated!
Use SoftwareSpec#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/gmp4-julia.rb:9:in `<class:Gmp4Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/gmp4-julia.rb:9:in `<class:Gmp4Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling BottleSpecification#sha1 is deprecated!
Use BottleSpecification#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/gmp4-julia.rb:15:in `block in <class:Gmp4Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling BottleSpecification#sha1 is deprecated!
Use BottleSpecification#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/gmp4-julia.rb:16:in `block in <class:Gmp4Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling SoftwareSpec#sha1 is deprecated!
Use SoftwareSpec#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:9:in `block in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:9:in `block in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:13:in `block (2 levels) in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:18:in `block (2 levels) in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:23:in `block (2 levels) in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:28:in `block (2 levels) in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling Resource#sha1 is deprecated!
Use Resource#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:33:in `block (2 levels) in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling BottleSpecification#sha1 is deprecated!
Use BottleSpecification#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:66:in `block in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!

Warning: Calling BottleSpecification#sha1 is deprecated!
Use BottleSpecification#sha256 instead.
/usr/local/Library/Taps/staticfloat/homebrew-julia/llvm33-julia.rb:67:in `block in <class:Llvm33Julia>'
Please report this to the staticfloat/julia tap!
```

**Comment by CodingAnarchy** [comment]
This also occurs when doing a normal `brew update && brew upgrade` cycle, which can be quite annoying.

**Comment by genos** [comment]
Came here to file the same issue; doing `brew update && brew upgrade`, as mentioned by @CodingAnarchy, gave the following:

<img width="838" alt="screen shot 2016-08-02 at 3 53 11 pm" src="https://cloud.githubusercontent.com/assets/884550/17342966/7df6f8f6-58c9-11e6-804d-35d59c2a79b5.png">

**Comment by genos** [comment]
It also shows up when calling `brew cleanup`

**Comment by staticfloat** [comment]
Thanks for the reports, this has been fixed by #215.
